### ISSUE TITLE
Blink/flash text

### ISSUE THREAD

**Issue by videosmith** [Cuperino/QPrompt#202]
Is it possible to use the (somewhat deprecated) Blink parameter found in older html code?
(Please see attached example)

[Blink.html.zip](https://github.com/Cuperino/QPrompt/files/12254533/Blink.html.zip)

**Comment by Cuperino** [comment]
I know there must be a way to achieve this, but I do not know it at the time.

Qt's HTML corresponds to subset of standard HTML 4.01. Blink was never a part of the HTML standard, just a feature Microsoft popularized through their [market dominance](https://en.wikipedia.org/wiki/Embrace,_extend,_and_extinguish). So it's not implemented in Qt. Having said that, Qt is so flexible I don't see why it couldn't be implemented, other than it being a ton of work.

**Comment by videosmith** [comment]
Is CSS supported?

**Comment by Cuperino** [comment]
Yes, but because Qt is not a browser, only CSS 2 selector types are supported, and not all styling options are implemented.
https://doc.qt.io/qt-5/stylesheet-syntax.html#selector-types

When writing HTML or CSS for Qt to parse, I do it in the style I would've done it back in 2010, and that typically gets me the results I'm looking for.

**Comment by videosmith** [comment]
Could selected text be "color cycled" between 2 colors at a specified duration?

**Comment by Cuperino** [comment]
Yes, and it is also possible to have multiple cursors at once, tho some refactoring would be required depending on how they were to be used.

**Comment by Cuperino** [comment]
Question, what are use-cases for this feature? By going from problem to solution instead of feature to solution, I might see solutions that were previously not as evident.

**Comment by videosmith** [comment]
Very limited, so not critical in any way for my usage.  Mostly as a method to alert live event speakers that they are running over their alloted time, changes in program cues, last minute housekeeping announcements etc.

**Comment by Cuperino** [comment]
I see!  It makes total sense to add blink.

What if in addition to the text's blink, I added a button that when pressed started oscillating the timer and the screen's borders between black and red? Other colors could also be used to indicate other kinds of feedback.

**Comment by videosmith** [comment]
Could be useful!

    On Saturday, August 5, 2023 at 12:09:37 PM EDT, Javier O. Cordero Pérez ***@***.***> wrote:  
 
 


I see! It makes total sense to add blink.

What if in addition to the text's blink, I added a button that when pressed started oscillating the timer and the screen's borders between black and red? Other colors could also be used to indicate other kinds of feedback.

**Comment by Liebeg** [comment]
> Very limited, so not critical in any way for my usage. Mostly as a method to alert live event speakers that they are running over their alloted time, changes in program cues, last minute housekeeping announcements etc.

If the text is blinking tho it makes it harder to read wich could lead to stuttering.
